src/schedule/ucc_schedule.c

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+#include "ucc_schedule.h"
+#include "utils/ucc_compiler_def.h"
+
+void ucc_event_manager_init(ucc_event_manager_t *em)
```

Comment:
return ucc_status_t

Reply:
why? it can't fail.

Reply:
yes, I understand it can't fail. I thought that if in future we add something more complicated here return code can be helpful. And same for other functions.